openff/interchange/interop/openmm/__init__.py

```diff
@@ -1,25 +1,21 @@
 """Interfaces with OpenMM."""
 from pathlib import Path
-from typing import Union
+from typing import TYPE_CHECKING, Union
 
 import openmm
-from openff.toolkit.topology import Topology
-from openff.units import unit as off_unit
-from openff.units.openmm import from_openmm as from_openmm_quantity
 from openmm import unit
 
-from openff.interchange.components.potentials import Potential
-from openff.interchange.constants import _PME
 from openff.interchange.exceptions import UnsupportedImportError
-from openff.interchange.interop.openmm._nonbonded import _process_nonbonded_forces
-from openff.interchange.interop.openmm._valence import (
-    _process_angle_forces,
-    _process_bond_forces,
-    _process_constraints,
-    _process_improper_torsion_forces,
-    _process_torsion_forces,
-)
-from openff.interchange.models import PotentialKey, TopologyKey
+
+__all__ = [
+    "to_openmm",
```

Comment:
I'm open to renaming this `to_openmm_system` or something

Reply:
cc: @Yoshanuikabundi

Reply:
I think that's probably a better name but it doesn't matter much.